scripts/haddocs.sh

```diff
@@ -92,3 +100,8 @@ for file in $(ls $OUTPUT_DIR/*/doc-index.json); do
     > /tmp/doc-index.json
   mv /tmp/doc-index.json "${OUTPUT_DIR}/doc-index.json"
 done
+
+# Copy modules map to output directory
+# TODO: dynamically generate
```

Comment:
Would it make sense to create an issue and address it rather soon?

Reply:
Definitely.